docs/reference/patches.rst

```diff
@@ -356,3 +356,12 @@ Python-formatted LMS settings in development. Values defined here override the v
 File: ``apps/openedx/settings/lms/production.py``
 
 Python-formatted LMS settings in production. Values defined here override the values from :patch:`openedx-lms-common-settings`.
+
+``uwsgi-config``
+================
+
+File: ``apps/openedx/settings/uwsgi.ini``
+
+A .INI formatted file used to extend or override the uWSGI configuration.
```

Comment:
Please add links to the uwsgi docs:
https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html#ini-files
https://uwsgi-docs.readthedocs.io/en/latest/Options.html